db/repo.cr

```diff
@@ -0,0 +1,14 @@
+require "pg"
+require "crecto"
+require "dotenv"
+
+env = Dotenv.load ".env"
+
+module Repo
+  extend Crecto::Repo
+
+  config do |conf|
+    conf.adapter = Crecto::Adapters::Postgres
+    conf.uri = env["POSTGRES_URL"]
```

Comment:
Use global `CONFIG` instance for env value so we have it centralized

Reply:
we'll stick with env for now as Bar has moved the global const to src/broken_crystals.cr